### ISSUE TITLE
How to fetch data for a keyword as a Company rather than a Keyword?

### ISSUE THREAD

**Issue by igalci** [GeneralMills/pytrends#437]
In the Google Trends auto-suggest, it offers to fetch the data for a keyword as a "keyword", "company", "topic", etc...

Is this possible with this API too? How to do that?

**Comment by flxai** [comment]
Yes, it is possible. You can use the website to get the topic's ID. Simply
search for your topic and choose the appropriate item. You should see something like 
`%2Fg%2F11c0vm752r` which you must translate to `keywords = ["/g/11c0vm752r"]`. See #423 for an example.

**Comment by cooperjaXC** [comment]
%2Fm%2F022hpx
